### PR TITLE
Follow-up to Describe imagesource inputs (PR#1445)

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1445

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh PTAL

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -952,20 +952,20 @@ a subdirectory within the archive, and if valid, the builder changes into that s
...
-and the indicated files/directories will be copied into the context directory
-of the build process.  This is the same directory into which the source repository
-content (if any) is cloned.  If the source path ends in `*/.*` then the content
-of the directory will be copied, but the directory itself will not be created
-at the destination.
+Additional files can be provided to the build process via images. Input images
+are referenced in the same way the `*From*` and `*To*` image targets are
+defined. This means both Docker images and image stream tags can be referenced.
+In conjunction with the image, you must provide one or more path pairs to
+indicate the path of the files/directories to copy out of the image and the
```
Comment (tnguyen-rh) [inline dev_guide/builds.adoc]: I was told somewhere that slash is undesirable.  Better something like "files and directories".

**Comment by tnguyen-rh** [comment]
@ahardin-rh One quibble, otherwise LGTM.

**Comment by ahardin-rh** [comment]
@tnguyen-rh Thanks! This is now edited.

**Comment by ahardin-rh** [comment]
revhistory in original PR https://github.com/openshift/openshift-docs/pull/1445

**Pull request merged**